src/wire/ipv6.rs

```diff
@@ -0,0 +1,382 @@
+use core::fmt;
+
+use byteorder::{ByteOrder, NetworkEndian};
+
+pub use super::IpProtocol as Protocol;
+
+// Helper used to convert 8 given u16 arguments to a [u8; 16]
+// slice
+macro_rules! write_to_addr {
```

Comment:
Do you really need a macro for something that will be used exactly once? Just expand it with copy/paste, it's more clear how it works then.

Reply:
Nah, just being lazy. Updated.